public/data/amehi0index.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Amelia hill",
```

I think it'd be best if you capitalize the `h` here 😅 What do you think? Thanks.
```suggestion
  "name": "Amelia Hill",
```

---

I think you are absolutely correct! I was sooo confused trying to figure everything out that I missed those. Thank you!